src/Cake/Program.cs

```diff
@@ -63,7 +64,14 @@ public async Task<int> Run(string[] args)
                 config.AddExample(new[] { "build.cake", "--tree" });
             });
 
-            return await app.RunAsync(args);
+            return await app.RunAsync(
+                args.Select(
+                    arg => arg switch {
+                        // Opt-out Spectre.Console.Cli no version parameter for default command
+                        "-v" => "--ver",
+                        "--version" => "--ver",
+                        _ => arg
+                    }));
```

Comment:
This is a terrible workaround, should've probably just addressed if first arg is `-v` / `--version`... but the problem goes deeper.

Spectre.Console.Cli switched from
```diff
-                if (firstArgument.Equals("--version", StringComparison.OrdinalIgnoreCase) ||
-                    firstArgument.Equals("-v", StringComparison.OrdinalIgnoreCase))
```
to
```diff
+        if (args.Contains("-v") || args.Contains("--version"))
+        {
```

default command behavior changed from first to any supplied argument is `--version` / `-v` will print the version and exit the application. even i.e.  `--` between command and remaining args used.

so will have to think about this a bit more. moving PR to WIP.